setup.py

```diff
@@ -28,7 +28,7 @@
     data_files=[('.', ['version.txt', 'CHANGELOG.md', 'sap_xssec.ini'])],
     test_suite='tests',
     install_requires=[
-        'requests>=2.21.0',
+        'requests>=2.22.0',
```

Comment:
i would stay on >= 2.21.0 because of version mismatch, see

https://2.python-requests.org/en/master/
vs.
https://3.python-requests.org/#

python3 uses https://3.python-requests.org/# , even if it seems that 2.22.0 is available

Reply:
okay, but according to https://pypi.org/project/requests/ it's available to python 3